### ISSUE TITLE
predict

### ISSUE THREAD

**Issue by gogogo617** [ZYangChen/Conv-fusion-Transformer-with-Distribution-Contrast-Loss#1]
Hello, thank you for your work.
May I ask, I only have a few satellite photos and their RPC files at the moment? Can I use them to predict altitude or generate DSM? Thank you very much!

**Comment by ZYangChen** [comment]
Yes, you can. However, DC-SatMVS is a fully-supervised method, so I recommend dividing the training, validation, and test sets on your dataset and retraining the model accordingly.
